chef-config/lib/chef-config/package_task.rb

```diff
@@ -185,10 +190,27 @@ def define
 #       pre-release versions like "10.14.0.rc.2".  Please use Rubygem's
 #       Gem::Version class instead.
 #
-        VERSION_RB
-        IO.write(version_rb_path, contents)
+          VERSION_RB
+          IO.write(version_rb_path, contents)
+        end
+
+        task :bump => %w{version:bump_patch version:update}
+
+        task :show do
+          puts version
+        end
+
+        # Add 1 to the current patch version in the VERSION file, and write it back out.
+        task :bump_patch do
+          current_version = version
+          new_version = current_version.sub(/^(\d+\.\d+\.)(\d+)/) { "#{$1}#{$2.to_i + 1}" }
```

Comment:
not a blocker but using Gem::Version to parse the patch and increment might be more clear.